docs/src/fs.rst

```diff
@@ -263,6 +266,8 @@ API
 
     .. versionchanged:: 1.20.0 `UV_FS_COPYFILE_FICLONE` is supported.
 
+    .. versionchanged:: 1.20.0 `UV_FS_COPYFILE_FICLONE_FORCE` is supported.
```

Comment:
Maybe include the new option in the same versionchanged tag as `UV_FS_COPYFILE_FICLONE`

Reply:
Good idea. I wasn't sure how long this PR would be hanging out for.